### PR TITLE
C105

### PR DESCRIPTION
**Pull request by SupriyaPamulapati**

### CODE REVIEW

File: src/test/java/test/resourceTest/ResourcesTest.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        tweetPost = mock(Controller.class);
+        }
+
+    @Test
+    public void testcase_check_EmptyPost() throws TwitterException {
+        when(restConfig.configurationBuilder()).thenReturn(new ConfigurationBuilder());
+        Twitter twitter = TwitterFactory.getSingleton();
+        String message = "ok";
+        Status status = twitter.updateStatus(message);
+        System.out.println("Successfully updated the status to [" + status.getText() + "].");
```
Comment (PoojaMohite-Khoros) [inline src/test/java/test/resourceTest/ResourcesTest.java:41]: Can u please assert the status here

File: src/test/java/test/resourceTest/ResourcesTest.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+    @Test
+    public void testcase_searchTweets() throws TwitterException {
+        when(restConfig.configurationBuilder()).thenReturn(new ConfigurationBuilder());
+        Twitter twitter = TwitterFactory.getSingleton();
+        ArrayList<String> arrayList = new ArrayList<String>();
+        Query query = new Query("source:twitter4j SupriyaChowdar9");
+        QueryResult result = twitter.search(query);
+        for (Status status : result.getTweets()) {
+            System.out.println("@" + status.getUser().getScreenName() + ":" + status.getText());
```
Comment (PoojaMohite-Khoros) [inline src/test/java/test/resourceTest/ResourcesTest.java:130]: Please assert here

**Review by PoojaMohite-Khoros — CHANGES_REQUESTED** [review]
Asserts are missing,
Add asserts and u are good to merge after adding asserts

File: src/test/java/test/resourceTest/ResourcesTest.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        ArrayList<String> str = new ArrayList<String>();
+        str.add("hii");
+        when(tweetPost.getTweets()).thenReturn(Response.ok(str).build());
+        ArrayList<String> arrayList = new ArrayList<String>();
+        arrayList.add("hii");
+        Response expectedTweet = Response.ok(arrayList).build();
+        Response actualTweet = tweetPost.getTweets();
+        Assert.assertEquals(expectedTweet.getStatus(), actualTweet.getStatus());
+        Assert.assertEquals(expectedTweet.getEntity(), actualTweet.getEntity());
+        System.out.println(expectedTweet);
```
Comment (raj8789) [inline src/test/java/test/resourceTest/ResourcesTest.java]: if assertEquals is used remove sout

File: src/test/java/test/resourceTest/ResourcesTest.java (hunk 2 above)
Comment (SupriyaPamulapati) [reply src/test/java/test/resourceTest/ResourcesTest.java:153]: iam still working on it its bit high end to understand this and add assert to it

**Review by raj8789 — COMMENTED** [review]
i have asked some code to review again

**Pull request merged**